src/index.html

```diff
@@ -6,9 +6,30 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#" class="img"><img src="/images/logo.png" alt="Moyo_logo"></a>
+        <div class="nav">
+          <nav>
+            <ul class="list"></ul>
+            <li class="header_nav item-1"><a href="#" class="is-active">Apple</a></li>
```

Comment:
Rewrite classes according BEM rules. Fix all cases 
And put all `li` tags inside `ul` tag
```suggestion
        <div>
          <nav class="nav">
            <ul class="nav__list">
              <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
            </ul>
            
```